test/bin/scenario.sh

```diff
@@ -60,13 +60,9 @@ prepare_kickstart() {
     local template="$2"
     local boot_commit_ref="$3"
 
-    local full_vmname
-    local output_file
-    local vm_hostname
-
-    full_vmname="$(full_vm_name "${vmname}")"
-    output_file="${SCENARIO_INFO_DIR}/${SCENARIO}/vms/${vmname}/kickstart.ks"
-    vm_hostname="${full_vmname/./-}"
+    local -r full_vmname="$(full_vm_name "${vmname}")"
+    local -r output_file="${SCENARIO_INFO_DIR}/${SCENARIO}/vms/${vmname}/kickstart.ks"
+    local -r vm_hostname="${full_vmname/./-}"
```

Comment:
It's easier to review PRs if we keep the changes to those that are strictly necessary.

Reply:
Agreed. I'm reverting these changes.